api/app/signals/apps/email_integrations/actions/abstract.py

```diff
@@ -51,17 +51,28 @@ def __call__(self, signal, dry_run=False):
 
         return False
 
-    def get_additional_context(self, signal, dry_run=False):
+    def mail_with_context(self, signal: Signal, dry_run=False, **kwargs):
```

Comment:
No need for this extra function `mail_with_context` please remove and use the `send_mail` function that is already in the class